<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubLayerTextPage</class>
 <widget class="QWidget" name="SubLayerTextPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="hasLabel">
     <property name="text">
      <string>Has Label</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="labelTagPick">
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Halo Size</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="color"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="fontWeight"/>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="haloColorPick">
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Halo Color</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="haloColor"/>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="maxWrapWidth">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>50000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="height"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="offsetLabel">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDoubleSpinBox" name="offset">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="haloSize"/>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="colorPick">
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="maxWrapWidthLabel">
     <property name="text">
      <string>Wrap Width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="text"/>
   </item>
   <item row="10" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Min Zoom</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="minZoom">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
